core/modules/catalog/helpers/tax/calculateProductTax.ts

```diff
@@ -0,0 +1,47 @@
+import updateProductPrices from './updateProductPrices'
+import getRate from './getRate'
+
+const calculateProductTax = ({ product, taxClasses, taxCountry = 'PL', taxRegion = '', sourcePriceInclTax = false, deprecatedPriceFieldsSupport = false, userGroupId }) => {
+  const rate = getRate({ taxClasses, product, taxCountry, taxRegion, userGroupId })
+
+  if (rate) {
+    updateProductPrices(product, rate, sourcePriceInclTax, deprecatedPriceFieldsSupport)
+    return
+  }
+
+  updateProductPrices(product, { rate: 0 })
```

Comment:
Same here we need to use the config value from `tax.sourcePriceIncludesTax`. 

`updateProductPrices(product, {rate: 0, finalPriceInclTax: currentStoreView().tax.sourcePriceIncludesTax})`

Reply:
same here: https://github.com/DivanteLtd/vue-storefront/blob/develop/core/modules/catalog/helpers/tax.ts#L253